cfme/tests/infrastructure/test_kubevirt.py

```diff
@@ -11,6 +14,21 @@
 ]
 
 
+@pytest.fixture
+def vm_name():
+    vm_name = random_vm_name('k6tvm')
+    return vm_name
+
+
+@pytest.fixture
+def test_vm(provider, vm_name):
```

Comment:
**REQUIRED:** I don't think this is going to pass the "a test name must be unique across all the tests" criteria.   Name should be changed to something like "test_kubevirt_vm"

Reply:
@jamesooden this function is a pytest fixture, not a test. And the `vm_name` is going to be random, so that the VM created on the provider would have a random name. From the test you're calling this fixture, would keep its name unique and pass the criteria.

Reply:
Ah got it.   That said, since tests typically start with test_ is it wise to name this fixture starting with test_?

Reply:
@jamesooden won't be a issue I think as its marked with a fixture, but I'll rename it for simplicity.